Find a functionally equivalent code:com.android.systemui.statusbar.policy.DeadZone.onTouchEvent:COMMENT
Method Modifier: public      
Comment:// I made you a touch event...

@@ -1,18 +1,18 @@
 {
     if (DEBUG) {
-        Log.v(TAG, this + " onTouch: " + MotionEvent.actionToString(event.getAction()));
+        Slog.v(TAG, this + " onTouch: " + MotionEvent.actionToString(event.getAction()));
     }
     final int action = event.getAction();
     if (action == MotionEvent.ACTION_OUTSIDE) {
         poke(event);
     } else if (action == MotionEvent.ACTION_DOWN) {
         if (DEBUG) {
-            Log.v(TAG, this + " ACTION_DOWN: " + event.getX() + "," + event.getY());
+            Slog.v(TAG, this + " ACTION_DOWN: " + event.getX() + "," + event.getY());
         }
         int size = (int) getSize(event.getEventTime());
         if ((mVertical && event.getX() < size) || event.getY() < size) {
             if (CHATTY) {
-                Log.v(TAG, "consuming errant click: (" + event.getX() + "," + event.getY() + ")");
+                Slog.v(TAG, "consuming errant click: (" + event.getX() + "," + event.getY() + ")");
             }
             if (mShouldFlash) {
                 post(mDebugFlash);

